src/adaptors/amqp/connection_manager.c

```diff
@@ -414,8 +428,17 @@ QD_EXPORT qd_connector_t *qd_dispatch_configure_connector(qd_dispatch_t *qd, qd_
         item->host_port = malloc(hplen);
         snprintf(item->host_port, hplen, "%s:%s", item->host , item->port);
 
-        DEQ_INSERT_TAIL(ct->conn_info_list, item);
+        //
```

Comment:
Should this also be filtered by role (i.e. don't create vanFlow records for 'normal' connectors)?

Reply:
Yes it should. I think we're getting lucky here - the CI test checks that tcpConnectors don't add LINK records but that's because the tcpConnector implementation is in the tcp adaptor and this code doesn't apply.

But yeah, better to be explicit here - good catch.